apps/balancer-tools/src/app/apr/round/(components)/PoolsCards.tsx

```diff
@@ -0,0 +1,58 @@
+import { ChevronRightIcon } from "@radix-ui/react-icons";
+
+import { fetcher } from "#/utils/fetcher";
+import { formatNumber } from "#/utils/formatNumber";
+
+interface DunePoolData {
+  symbol: string;
+  pct_votes: number;
+  votes: string;
+}
+
+function PoolCard({
+  data: { symbol, pct_votes: pctVotes, votes },
+}: {
+  data: DunePoolData;
```

Comment:
Any reason why we didn't import it from lib/dune.ts?

Reply:
No 😆 I didn't kow we had this there tbh